packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -1087,6 +1087,11 @@ export interface ProcessTaskDto extends TaskDto, CommandProperties {
     windows?: CommandProperties;
 }
 
+export interface LanguagesInfo {
```

Comment:
I'm still not sure about naming, it seems to be more appropriate to rename to `PluginInfo` and keep `id` and `name` as before.

You can pass these information then for other rpc service, not only for languages.